doc/source/user/plugins.rst

```diff
@@ -335,6 +335,57 @@ As with other plugins, :class:`~watts.PluginABCE` is easily used by::
+
+    Scenario	FUEL	Type	Unit	2022	2023	2024	2025	2026	2027	2028	2029	2030	2031	2032	2033	2034	2035	2036	2037	2038	2039	2040	2041	2042	2043	2044	2045	2046	2047	2048	2049	2050	2051	2052	2053	2054	2055	2056	2057	2058	2059	2060
+    Reference	Coal	Real	2021 $/MMBtu	1.64	1.61	1.65	1.57	1.52	1.50	1.49	1.50	1.50	1.50	1.50	1.50	1.51	1.52	1.53	1.54	1.56	1.56	1.58	1.59	1.60	1.60	1.62	1.63	1.64	1.65	1.65	1.66	1.66	1.75	1.85	1.96	2.07	2.19	2.31	2.44	2.58	2.73	2.89
+    Reference	NG	Real	2021 $/MMBtu	3.84	3.49	3.17	3.00	2.98	3.08	3.25	3.36	3.46	3.54	3.58	3.65	3.64	3.64	3.65	3.67	3.68	3.69	3.72	3.73	3.70	3.71	3.65	3.61	3.60	3.60	3.62	3.60	3.59	3.80	4.01	4.24	4.48	4.74	5.01	5.29	5.59	5.91	6.25
+    Reference	Petroleum	Real	2021 $/MMBtu	12.18	10.50	11.39	11.61	11.89	12.20	12.46	12.58	12.82	13.08	13.27	13.42	13.54	13.67	13.86	14.03	14.20	14.26	14.55	14.70	14.75	15.00	15.28	15.36	15.58	15.65	15.62	15.65	15.59	16.48	17.42	18.41	19.46	20.57	21.74	22.98	24.29	25.68	27.14
+    Reference	Nuclear	Real	2021 $/MMBtu	0.72	0.72	0.72	0.72	0.72	0.73	0.73	0.73	0.73	0.73	0.73	0.74	0.74	0.74	0.74	0.74	0.74	0.75	0.75	0.75	0.75	0.76	0.76	0.76	0.76	0.76	0.77	0.77	0.77	0.81	0.86	0.91	0.96	1.02	1.07	1.14	1.20	1.27	1.34
+    Reference	Biomass	Real	2020 $/MMBTU	5.00	5.11	5.22	5.34	5.45	5.57	5.70	5.82	5.95	6.08	6.22	6.35	6.49	6.63	6.78	6.93	7.08	7.24	7.40	7.56	7.73	7.90	8.07	8.25	8.43	8.61	8.80	9.00	9.20	9.72	10.27	10.86	11.48	12.13	12.82	13.56	14.33	15.15	16.01
+    Reference	OGS	Real	2020 $/MMBTU	3.84	3.49	3.17	3.00	2.98	3.08	3.25	3.36	3.46	3.54	3.58	3.65	3.64	3.64	3.65	3.67	3.68	3.69	3.72	3.73	3.70	3.71	3.65	3.61	3.60	3.60	3.62	3.60	3.59	3.80	4.01	4.24	4.48	4.74	5.01	5.29	5.59	5.91	6.25
+    Base	Coal	Real	2020 $/MMBTU	1.57	1.54	1.57	1.50	1.45	1.43	1.43	1.43	1.43	1.44	1.43	1.43	1.44	1.45	1.46	1.47	1.49	1.49	1.51	1.52	1.52	1.53	1.54	1.56	1.56	1.57	1.57	1.58	1.58	1.67	1.77	1.87	1.98	2.09	2.21	2.33	2.47	2.61	2.76
+    Base	NG	Real	2020 $/MMBTU	3.67	3.34	3.03	2.87	2.84	2.94	3.10	3.21	3.30	3.38	3.42	3.48	3.48	3.47	3.48	3.50	3.52	3.52	3.55	3.56	3.54	3.54	3.48	3.45	3.44	3.43	3.46	3.44	3.43	3.62	3.83	4.05	4.28	4.52	4.78	5.05	5.34	5.65	5.97
+    Base	Petroleum	Real	2020 $/MMBTU	11.63	10.03	10.88	11.09	11.35	11.65	11.90	12.01	12.24	12.49	12.68	12.82	12.93	13.05	13.24	13.40	13.56	13.61	13.90	14.04	14.09	14.32	14.60	14.67	14.88	14.95	14.92	14.94	14.89	15.74	16.63	17.58	18.58	19.64	20.76	21.95	23.20	24.52	25.92
+    Base	Nuclear	Real	2020 $/MMBTU	{{fuel_2022}}	{{fuel_2023}}   {{fuel_2024}}	{{fuel_2025}}	{{fuel_2026}}	{{fuel_2027}}	{{fuel_2028}}	{{fuel_2029}}	{{fuel_2030}}	{{fuel_2031}}	{{fuel_2032}}	{{fuel_2033}}	{{fuel_2034}}	{{fuel_2035}}	{{fuel_2036}}	{{fuel_2037}}	{{fuel_2038}}	{{fuel_2039}}	{{fuel_2040}}	{{fuel_2041}}	{{fuel_2042}}	{{fuel_2043}}	{{fuel_2044}}	{{fuel_2045}}	{{fuel_2046}}	{{fuel_2047}}	{{fuel_2048}}	{{fuel_2049}}	{{fuel_2050}}	{{fuel_2051}}	{{fuel_2052}}	{{fuel_2053}}	{{fuel_2054}}	{{fuel_2055}}	{{fuel_2056}}	{{fuel_2057}}	{{fuel_2058}}	{{fuel_2059}}	{{fuel_2060}}
```

Comment:
I recommend we skip the large table - only show the line of interest with 1 or 2 additional lines for information.
For the nuclear template - it can be improved by leveraging Jinja ability to use list:
```
{%for fuel_yr in list_fuel_yr %} 
    {{fuel_yr}} 
{% endfor %}
```
The python input will be something like:
```
list_fuel_yr = [0.4, 0.5, 0.6...]
```